react/hooks/useInterval.spec.js

```diff
@@ -0,0 +1,68 @@
+import useInterval from './useInterval'
+import { renderHook } from '@testing-library/react-hooks'
+
+async function wait(duration) {
+  console.log('setTimeout', duration)
+  return new Promise(resolve => {
+    window.setTimeout(() => {
```

Comment:
Could we use [timer mocks here](https://jestjs.io/docs/en/timer-mocks.html)?

Reply:
Vu ensemble, pas adapté pour ce cas